tasks.py

```diff
@@ -0,0 +1,646 @@
+from invoke import task, run
+import platform
+import os
+
+PLATFORM = platform.system()
+MESSAGE_FAILED = 'failed to install. Please, use "invoke task_name --verbose" to check out the stderr and stdout responses.'
+MESSAGE_OK = "successfully installed"
+MESSAGE_WRONG_PLATFORM = "Unsupported Platform. Only Ubuntu (16.04+), Debian (9+), Mac OS. Your system is {}".format(
+    platform.platform()
+)
+MESSAGE_ALREADY_INSTALLED = "already installed"
+SERVE_CELERY = "celery worker --app=project --loglevel=INFO -Q builds,email,cleanup"
+SERVE_HTTPS_SERVER = "python manage.py runsslserver --certificate certs/local_lookit.mit.edu.pem --key certs/local_lookit.mit.edu-key.pem"
+SERVE_HTTP_SERVER = "python manage.py runserver"
+SERVE_NGROK = "ngrok http https://127.0.0.1:8000"
+BASE_DIR = os.getcwd()
+PATH_TO_CERTS = os.path.join(BASE_DIR, "certs")
+
+
+@task
+def system_setup(c, verbose=False):
+    """
+
+    1. Install pipenv, brew, docutils, and celery
+    2. Set the debug to False in a local setting file for serving locally.
+
+    usage: invoke system-setup or invoke system-setup --verbose
+
+    """
+
+    run("echo '*** SYSTEM SETUP ***'")
+    run("echo 'Installing pipenv, brew, and docutils & Configuring local settings'")
+    packages = ("pipenv", "docutils", "celery")
+
+    for package in packages:
+
+        if run("pip install {}".format(package), hide=not verbose, warn=True).ok:
+
+            run('echo "===>{} {}"'.format(package, MESSAGE_OK))
+
+        else:
+
+            run('echo "===>{} {}"'.format(package, MESSAGE_FAILED))
+
+    # installing linuxbrew
+    if PLATFORM == "Linux":
+        run(
+            'sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            "test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)",
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            "test -d /home/linuxbrew/.linuxbrew && eval \$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)",
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            'test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.bash_profile',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            'echo "eval $($(brew --prefix)/bin/brew shellenv)"',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            "sudo apt-get install build-essential curl git", hide=not verbose, warn=True
+        )
+        if run("command -v brew", hide=not verbose, warn=True).ok:
+            run('echo "===>brew {}"'.format(MESSAGE_OK))
+        else:
+            run('echo "===>brew {}"'.format(MESSAGE_FAILED))
+
+    if PLATFORM == "Darwin":
+        if run("command -v cask ", hide=not verbose, warn=True).ok:
```

Comment:
Won't this always fail? `cask` isn't the executable, `brew` is

Reply:
Cask is actually already in brew, so I do not need to check whether I have it. All I need to do is to make sure that I have "brew."

Reply:
Just to make sure it is there, I will "brew tap," which is a way of adding new repositories for brew to install or keep track off.